local/include/core/ProxyVerifier.h

```diff
@@ -547,29 +697,14 @@ class AbsenceCheck : public RuleCheck
   bool _expects_duplicate_fields = false;
 };
 
-class ContainsCheck : public RuleCheck
+class SubstrCheck : public CompareCheck
 {
 public:
-  ~ContainsCheck() { }
-
-  /** Construct @a ContainsCheck with a given name and "contains" value.
-   *
-   * @param name The name of the target field
-   * @param value The associated "contains" value with the target field,
-   * that is used with strcasecmp comparisons
-   */
-  ContainsCheck(swoc::TextView name, swoc::TextView value);
+  ~SubstrCheck() { }
```

Comment:
Don't bother with this if there's no body.

Reply:
Use ` = default;` if you want to be explicit.